NUAttribute.js

```diff
@@ -102,8 +102,14 @@ export default class NUAttribute extends NUObject {
                 dataTypeMismatch = false;        
             if (attrObj.subType === NUAttribute.ATTR_TYPE_INTEGER || attrObj.subType === NUAttribute.ATTR_TYPE_FLOAT) {
                 dataTypeMismatch = (typeof listElementValue !== 'number');
-            } else if (attrObj.subType !== NUAttribute.ATTR_TYPE_ENUM && typeof listElementValue === 'object' && !(listElementValue instanceof attrObj.subType)) {
-                dataTypeMismatch = true;
+            } else if (attrObj.subType !== NUAttribute.ATTR_TYPE_ENUM && typeof listElementValue === 'object') {
+                if (attrObj.subType.prototype && attrObj.subType.prototype.constructor.name) {
```

Comment:
The constructor name is not guaranteed on minimization!

Reply:
@natabal addresed comment. using getClassName now. can you pls review?